front-end/src/components/Content/User/User.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import "./style.scss";
+
+export default function User() {
+  const data = [
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "true",
+    },
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "true",
+    },
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "true",
+    },
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "true",
+    },
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "true",
+    },
+  ];
+
+  return (
+    <table>
+      <tbody>
+        <tr>
+          <th class="checkbox-Area">
+            <input type="checkbox" name="scales" />
+          </th>
+          <th class="title_Area">Title</th>
+          <th class="username_Area">UserName</th>
+          <th class="email_Area">Email</th>
+          <th class="status_Area">Status</th>
+          <th>Title</th>
+        </tr>
+        {data.map((item) => {

```

Comment:
@Taiga17 -san
Good job!.

Add note more when we loop a list we need to set a **key** for each item, to make sure each row is unique.
Please refer document: https://reactjs.org/docs/lists-and-keys.html#keys
Thank you.

Reply:
@duyphamsalto 
Your suggestion is more beautiful.
I will change the code.
Thank you for the code review.